### PR TITLE
Merge private & public repositories.

### PR DESCRIPTION
**Pull request by dwfmarchant**
Merges the changes that were made on the master branches of the private and the public repositories.    Adds a CSEM example that runs on a tensor mesh.

### CODE REVIEW

File: inversion-examples/testCSEMSingleMesh/runMaxFreqUBCInversionSingleMesh.jl [hunk 1]
```diff
@@ -66,105 +67,92 @@ for i = 1:length(rcv)
...
 # receivers
 Receivers = spzeros(Complex128, sum(M.ne), length(rcv))
 for i = 1:length(rcv)
-        ply            = [rcv[i][:,1:2] zSurf*ones(size(rcv[i],1))]
-        if sum(ply[1,:]-ply[2,:]) == 0.0
-          ply[1,3] = zSurf-10
-          ply[2,3] = zSurf+10
-        end
-	Receivers[:,i] = complex(getEdgeIntegralOfPolygonalChain(M,ply,normalize=true))
+	Receivers[:,i] = complex(getEdgeIntegralOfPolygonalChain(M,rcv[i],nEx,nEy,nEz,normalize=true))
```
Comment (Pbellive) [inline inversion-examples/testCSEMSingleMesh/runMaxFreqUBCInversionSingleMesh.jl]: Now that Roman's JOcTree changes have been merged, could you update the `getEdgeIntegralOfPolygonalChain` calls to the new signature before merging? Thanks.

**Review by Pbellive — CHANGES_REQUESTED** [review]
Hi Dave,
                 Thanks a lot for getting this ready. I'd like to make sure this runs with the latest updates to JOcTree. The only thing I noticed was one spot where the call signature for `getEdgeIntegralOfPolygonalChain` had to be changed in the OcTree inversion examples. If you could fix that then I'll merge these changes. Thanks

**Comment by dwfmarchant** [comment]
I think that I did that already but I'll check tomorrow.

**Comment by dwfmarchant** [comment]
I think that's everything - I just reran the inversion examples and they all run again.

**Review by Pbellive — APPROVED** [review]

**Comment by Pbellive** [comment]
Merged. Thanks again Dave.

**Pull request merged**